lib/elixir/test/elixir/record_test.exs

```diff
@@ -262,6 +263,15 @@ defmodule RecordTest do
     end
   end
 
+  test "duplicate keys" do
+    assert capture_io(:stderr, fn ->
```

Comment:
Capturing a named device other than `:stdio`, such as `:stderr`, happens globally and requires `async: false`.

In this case, I would move the test to the `warning_test.exs` file.

Reply:
Ah, right. Fixed!